react/src/components/tooltip/SprkTooltip.js

```diff
@@ -0,0 +1,173 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import SprkIcon from '../icons/SprkIcon';
+import uniqueId from 'lodash/uniqueId';
+
+class SprkTooltip extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isToggled: this.props.isDefaultToggled || false
```

Comment:
I understand isDefaultToggled but isDefaultOpen is more clear and is what we use in the Toggle component as well.

Reply:
I changed it from isDefaultOpen to isDefaultToggled because the tooltip can be open without being toggled.